packages/lit/src/element/lit-root.ts

```diff
@@ -29,6 +29,9 @@ import type { WidgetElement } from './widget-element.js';
 
 @customElement('block-suite-root')
 export class BlockSuiteRoot extends WithDisposable(ShadowlessElement) {
+  @property({ attribute: false })
+  mode!: 'page' | 'edgeless';
```

Comment:
Yeah, we add property like `mode` in lit root. It's for building universal blocksuite editors instead of for affine app. I think we have already have this attribute in `editorContainer`. You can use something like `getEditorContainer(page).mode`. See: https://github.com/toeverything/blocksuite/blob/bb4a1efc37cf1c04bc11ce1445a74d0d507f0fb4/packages/blocks/src/_common/utils/query.ts#L155